rebasehelper/tests/test_utils.py

```diff
@@ -42,6 +44,77 @@
 from rebasehelper.utils import LookasideCacheHelper
 
 
+class TestGitHelper(object):
+
+    def write_config_file(self, config_file, name, email):
+        with open(config_file, 'w') as f:
+            f.write('''
+[user]
+    name = {0}
+    email = {1}
+'''.format(name, email))
```

Comment:
How about this? So it doesn't break indentation.
```python
                    f.write('[user]\n'
                            '    name = {0}\n'
                            '    email = {1}\n'.format(name, email))
```

Or do it your way but define it as constants at the top of the file, where indentation is not needed.

Reply:
Sure, I am fine for your suggestion. Just moment. I will rebase again. Same for `[include]` part.